src/main/java/com/google/sps/servlets/ContestsServlet.java

```diff
@@ -82,10 +84,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       }
     }
 
+    for (Long referendumId : electionReferendumIds) {
+      Key currentKey = KeyFactory.createKey(Referendum.ENTITY_NAME, referendumId.longValue());
+      try {
+        Entity currentReferendumEntity = datastore.get(currentKey);
+        referendumList.add(Referendum.fromEntity(currentReferendumEntity));
+      } catch (EntityNotFoundException e) {
+        response.setContentType("text/html");
+        response
+            .getWriter()
+            .println("Referendum with Id " + referendumId.toString() + " was not found.");
+        response.setStatus(400);
+        return;
+      }
+    }
+
     Gson gson = new Gson();
-    String json = gson.toJson(contestList);
+    String contestJson = gson.toJson(contestList);
+    String referendumJson = gson.toJson(referendumList);
 
     response.setContentType("application/json;");
-    response.getWriter().println(json);
+    response
+        .getWriter()
+        .println(
+            "{\""
```

Comment:
I feel like it's kind of confusing that the /contests servlet returns both "contests" and "referendums", but I don't have a better naming suggestion haha.

I think it's worth thinking about for maybe a few minutes, but if you can't think of any better naming schemes, I'm ok with submitting as is.

Reply:
Hmm to be honest it might make more sense to rename the Contest object, that way it stays consistent with the API

Reply:
I'm ok with that too :)